preseed/provision.erb

```diff
@@ -141,7 +141,8 @@ d-i grub-installer/only_debian boolean true
 d-i grub-installer/with_other_os boolean true
 <% if @host.params['install-disk'] -%>
 d-i grub-installer/bootdev string <%= @host.params['install-disk'] %>
-<% else -%>
+<% end -%>
+<% if @host.operatingsystem.name == 'Debian' and @host.operatingsystem.major.to_i == 8 -%>
```

Comment:
We'll really this, when it's released. Maybe someone opens a bug against the Ubuntu installer and the Debian change gets imported (I hope ;)).